Dockerfile

```diff
@@ -0,0 +1,34 @@
+# image for pg-elixir-tools
```

Comment:
it should be `Dockerfile.dev` I believe

Reply:
Why `.dev`? In this case we don't build a Docker image in ECR, but just use it for development purposes.
As there's no gonna be the possibility of having two different Docker images for local and other environments, with the Docker file should be enough.